repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthHandler.scala

```diff
@@ -1,11 +1,13 @@
 package org.openrepose.filters.rackspaceidentitybasicauth
```

Comment:
I removed them because they were flagging in IntelliJ IDEA's static analysis tools (e.g. FindBugs, PMD, CheckStyle) with the following:

    This inspection reports redundant enclosing blocks around expressions (including ones withing interpolated strings).

I'm not opposed to putting them back, I was the one the put them there initially. I was just clearing up noise in the IDE. If we do want them in, then we should update the rules for the Style Plugins and auto formatting too.